static/src/stylesheets/module/_adslot.scss

```diff
@@ -466,12 +466,12 @@
     }
 }
 .ad-slot--im {
-    box-sizing: border-box;
-    max-width: 50%;
     float: left;
+    margin: 5px $gs-gutter $gs-baseline 0;
```

Comment:
My only concern is if it is supposed to use the same styles why we just don't use the same class as rich links are? Do we need to abstract some class to something like `'.inlinebox` and move there these margin styles and reuse it?